_PLANS.md

```
@@ -0,0 +1,152 @@
+## UX Review & Roadmap (_ux-redesign-dashboard_)
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)